src/plugins/data/server/search/strategies/ese_search/ese_search_strategy.test.ts

```diff
@@ -93,37 +100,49 @@ describe('ES search strategy', () => {
         mockSubmitCaller.mockResolvedValueOnce(mockAsyncResponse);
 
         const params = { index: 'logstash-*', body: { query: {} } };
-        const esSearch = await enhancedEsSearchStrategyProvider(mockLegacyConfig$, mockLogger);
+        const esSearch = await enhancedEsSearchStrategyProvider(
+          mockLegacyConfig$,
+          mockSearchConfig,
+          mockLogger
+        );
 
         await esSearch.search({ params }, {}, mockDeps).toPromise();
 
         expect(mockSubmitCaller).toBeCalled();
         const request = mockSubmitCaller.mock.calls[0][0];
         expect(request.index).toEqual(params.index);
         expect(request.body).toEqual(params.body);
-        expect(request).toHaveProperty('keep_alive', '1m');
+        expect(request).toHaveProperty('keep_alive', '60000ms');
       });
 
       it('makes a GET request to async search with ID', async () => {
         mockGetCaller.mockResolvedValueOnce(mockAsyncResponse);
 
         const params = { index: 'logstash-*', body: { query: {} } };
-        const esSearch = await enhancedEsSearchStrategyProvider(mockLegacyConfig$, mockLogger);
+        const esSearch = await enhancedEsSearchStrategyProvider(
+          mockLegacyConfig$,
+          mockSearchConfig,
+          mockLogger
+        );
 
         await esSearch.search({ id: 'foo', params }, {}, mockDeps).toPromise();
 
         expect(mockGetCaller).toBeCalled();
         const request = mockGetCaller.mock.calls[0][0];
         expect(request.id).toEqual('foo');
         expect(request).toHaveProperty('wait_for_completion_timeout');
-        expect(request).toHaveProperty('keep_alive', '1m');
+        expect(request).toHaveProperty('keep_alive', '60000ms');
```

Comment:
all the `1m` -> `60000ms` in tests are because previously `1m` was hardcoded as a string literal, but now we use dynamic: `moment.duration(durationStringFromConfig).asMilliseconds`